src/pages/careers/_dept-layout-components/_zoho-job.tsx

```diff
@@ -0,0 +1,169 @@
+            display: none !important;
+        }
+        #rec_job_listing_div:not(#rec_job_listing_div *) {
+            text-align: center;
+            color: var(--color-grey-5);
+            font-weight: bold;
+            font-size: 2.4rem;
+            line-height: 3.6rem;
+            width: 100%;
+            display: block;
+        }
+        .rec_job_listing_div_jobs {
+            text-align: left;
+        }
+        ul.rec-job-info {
+            position: relative;
+            margin-bottom: 0 !important;
+            padding: 24px 35px 24px 18px;
+            width: 100%;
+            display: flex;
+            border-bottom: 1px solid #f2f3f4 !important;
+            align-items: center;
+
+            &::after {
+                position: absolute;
+                content: url(${RightArrow});
+                top: 50%;
+                right: 0;
+                font-size: 16px;
+                transform: translateY(-50%);
+            }
+        }
+        .embed_jobs_with_style .rec-group li.rec-job-title {
+            max-width: 450px;
+            width: 100%;
+            margin-bottom: 0;
+
+            @media (max-width: 767px) {
+                max-width: 50%;
+            }
+        }
+        div#rec_job_listing_div > div:first-child,
+        .zrsite_Industry,
+        .zrsite_Job_Type,
+        .zrsite_Job_Description,
+        .zrsite_Date_Opened,
+        .zrsite_Remote_Job,
+        .zrsite_State,
+        .zrsite_Work_Experience {
+            display: none !important;
+        }
+        .embed_jobs_head {
+            background: none;
+            padding: 0;
+        }
+        .embed_jobs_head2 {
+            background-color: none;
+            border: none;
+            padding: 0;
+            width: 100%;
+        }
+        .embed_jobs_with_style .rec-job-info a {
+            font-family: 'IBM Plex Sans', sans-serif !important;
+            color: #333333;
+            text-decoration: none;
+            font-weight: bold;
+            font-size: 2.4rem;
+            line-height: 3.6rem;
+            width: 100%;
+        }
+        li.zrsite_City,
+        li.zrsite_Country {
+            text-align: left;
+            color: var(--color-black-3);
+            line-height: 1.5;
+            max-width: 274px;
+            padding: 15px;
+            font-weight: 400;
+            font-size: 1.6rem;
+            width: auto !important;
+
+            span:first-child {
+                display: none !important;
+            }
+        }
+        li.zrsite_Country {
+            span:last-child::before {
+                position: relative;
+                content: ', ';
+            }
+        }
+    `
+
+    return (
+        <ZohoWrapper>
+            <Helmet>
+                <link
+                    rel="stylesheet"
+                    href="https://css.zohostatic.com/recruit/embed_careers_site/css/v1.0/embed_jobs.css"
+                    type="text/css"
+                />
```

Comment:
Can we not use the css that is coming from them and since you are already styling it yourself?